students/JerryH/Lesson03/list_lab.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+fruits = ['Apples', 'Pears', 'Oranges', 'Peaches']
+#Series 1
+def series1():
+    print(fruits)
+    addFruit = input("What fruit do you want to add?\n")
+    fruits.append(addFruit)
+    print(fruits)
+    num = int(input("Enter a number:\n"))
+    print(num)
+    print("The fruit is {}".format(fruits[num-1]))
```

Comment:
something to keep in mind is possible `IndexError` here in case the user enters 9 as example